homeassistant/components/file/config_flow.py

```diff
@@ -0,0 +1,126 @@
+"""Config flow for file integration."""
+
+import os
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import (
+    CONF_FILE_PATH,
+    CONF_FILENAME,
+    CONF_NAME,
+    CONF_PLATFORM,
+    CONF_UNIT_OF_MEASUREMENT,
+    CONF_VALUE_TEMPLATE,
+    Platform,
+)
+from homeassistant.helpers.selector import (
+    BooleanSelector,
+    BooleanSelectorConfig,
+    TemplateSelector,
+    TemplateSelectorConfig,
+    TextSelector,
+    TextSelectorConfig,
+    TextSelectorType,
+)
+
+from .const import CONF_TIMESTAMP, DEFAULT_NAME, DOMAIN
+
+BOOLEAN_SELECTOR = BooleanSelector(BooleanSelectorConfig())
+TEMPLATE_SELECTOR = TemplateSelector(TemplateSelectorConfig())
+TEXT_SELECTOR = TextSelector(TextSelectorConfig(type=TextSelectorType.TEXT))
+
+FILE_SENSOR_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_NAME): TEXT_SELECTOR,
+        vol.Required(CONF_FILE_PATH): TEXT_SELECTOR,
```

Comment:
Should we harmonize so it's same for sensor and notify e.g. `file_path` or `filename`

Reply:
done, we now use file_path, after import any valid dir can be used.